### PR TITLE
Add UFQFPN-32-1EP_5x5mm_P0.5mm_EP3.5x3.5mm.

### PR DESCRIPTION
**Pull request by piotr-wiszowaty**

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+UFQFPN-32-1EP_5x5mm_P0.5mm_EP3.5x3.5mm:
+  device_type: 'UFQFPN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.st.com/resource/en/datasheet/stm32g071k8.pdf'
+  ipc_class: 'ufqfpn' # 'qfn_pull_back'
```
Comment (cpresser) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml:6]: That is wrong. It is either `qfn_pull_back` or not set all all.
Please remove that line.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    paste_avoid_via: False
+
+  pitch: 0.50
+  num_pins_x: 8
+  num_pins_y: 8
+  chamfer_edge_pins: 0.09
+  #pin_count_grid:
+  #pad_length_addition: 0.5
+  #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
+  #include_suffix_in_3dpath: 'False'
```
Comment (cpresser) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml:58]: You can remove `chamfer_edge_pins`. The recommended layout only has this on Pin1.

Please also remove the unused lines here and above.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.st.com/resource/en/datasheet/stm32g071k8.pdf'
+  ipc_class: 'ufqfpn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 5
+  body_size_y:
+    nominal: 5
```
Comment (cpresser) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml:12]: There is tolerance for that in the datatsheet. Please add that
```yaml
body_size_x: 5 +/-0.1
body_size_y: 5 +/-0.1
```

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
+    #paste_between_vias: 1
+    #paste_rings_outside: 1
+    EP_paste_coverage: 0.75
```
Comment (cpresser) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml:46]: I don't see a recommendation for paste coverage in the datatsheet. Did I miss it?
If no, remove that line, then the value defaults to 0.65 which is recommended by IPC.

**Review by cpresser — CHANGES_REQUESTED** [review]

**Comment by cpresser** [comment]
One more thing: can you please remove the  commented out lines. They just add noise. Further contributors will just copy them and add more noise. IMHO this is cargo cult and should stop.

**Comment by piotr-wiszowaty** [comment]
Sure. Done.

**Comment by cpresser** [comment]
Thanks!

**Review by cpresser — APPROVED** [review]

**Pull request merged**